MANIFEST.in

```diff
@@ -39,6 +39,7 @@ exclude Dockerfile
 exclude mypy.ini
 exclude sytest-blacklist
 exclude test_postgresql.sh
+exclude .deepsource.toml
```

Comment:
I'm not sure it makes sense to include this config file in our repository unless it was something we were depending on.

Reply:
I included the file by mistake in the Manifest since the CI gave a hint to do so.
I have now excluded the file from the Manifest.
The .deepsource.toml file is not needed to be bundled in the generated files.

Reply:
Right, but we're not goign to run this in CI, so please remove the `.deepsource.toml` file and this change. Thanks!

Reply:
> then it's better to just reject this PR

Well the fixes in here are good, so it doesn't make sense to reject the PR.

Reply:
> Right, but we're not goign to run this in CI, so please remove the `.deepsource.toml` file and this change. Thanks!

I have removed  `.deepsource.toml` file  and updated the PR.
Thanks @clokep